### DISCUSSION TITLE
Complicated web page in Autoconnect

### DISCUSSION THREAD

**Discussion by Warlib1975** [Hieromon/AutoConnect#460] [How can I?]
Hi, 
 
We would like to create a complicated web form to manage a timetable for the ESP32 chip. Visually it should look like this: 
![image](https://user-images.githubusercontent.com/11045291/153694564-dd65d45d-f9b3-4bf0-a0bd-5599027c7a64.png)

Would you recommend the best approach how to code it? Do you have some examples of how to create such kind of form where a lot of elements should be placed in specific places of the webform?

**Comment by Hieromon** [comment]
Your grid has 7 columns, but it's a repeat of SUN, MON. Is it a mistake in the indicator for the day of the week from Sunday to Saturday?

**Comment by Warlib1975** [comment]
Hi. 
it's my mistake with columns name. The right one is
![image](https://user-images.githubusercontent.com/11045291/153742491-77363eff-efc6-4bfe-af43-453bdf7459f6.png)

**Reply by Hieromon** [reply]
It is not recommended to use AutoConnectElemets heavily. It is not suitable for properties where similar elements are repeated. (Cannot be arrayed)
It should be processed natively with raw HTML. How about this? https://codepen.io/marclundgren/pen/NWxoNY
